java/core/pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>com.google.protobuf</groupId>
     <artifactId>protobuf-parent</artifactId>
-    <version>3.7.0</version>
+    <version>3.7.0-rc1</version>
```

Comment:
3.7.0-rc-1

Reply:
I actually already pushed the artifacts for this one so I think it's too late to change for Java.